backend/src/packages/chat-messages/libs/types/chat-message-create-data.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type ChatMessageCreateRequestDto } from './types.js';
+
+type ChatMessageCreateData = ChatMessageCreateRequestDto & {
```

Comment:
let's rename it to ChatMessageCreatePayload
```suggestion
type ChatMessageCreatePayload = ChatMessageCreateRequestDto & {
```